### PR TITLE
Fix config

### PR DESCRIPTION
**Pull request by victorjzsun**

### CODE REVIEW

**Review by Yuxuan-Lin — APPROVED** [review]

**Review by rg3515 — APPROVED** [review]
lgtm

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (haohao1331) [inline .vscode/settings.json:1]: shouldn't the entire .vscode folder be in .gitignore?

File: Minor-Changes Ahead [hunk 2]
```diff
@@ -26,4 +26,4 @@
 5. General
     a. Password Changes
     b. FAQ
-    c. Upload
\ No newline at end of file
+    c. Upload
```
Comment (haohao1331) [inline Minor-Changes Ahead:29]: why this change?

**Review by haohao1331 — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (victorjzsun) [reply .vscode/settings.json:1]: We need our vscode settings to be consistent for all members
